pkg/crc/preflight/preflight_daemon_task_check_windows.go

```diff
@@ -71,6 +73,10 @@ func checkIfDaemonTaskInstalled() error {
 }
 
 func fixDaemonTaskInstalled() error {
+	// Remove older task if exist
```

Comment:
`if it exists`, but the comment is not really useful, the method name is already telling as much